lib/model/model.go

```diff
@@ -288,11 +291,6 @@ func (m *model) startFolderLocked(cfg config.FolderConfiguration) {
 		}
 	}
 
-	// Close connections to affected devices
-	m.fmut.Unlock()
-	m.closeConns(cfg.DeviceIDs(), fmt.Errorf("started folder %v", cfg.Description()))
-	m.fmut.Lock()
-
 	v, ok := fset.Sequence(protocol.LocalDeviceID), true
```

Comment:
the ok is always true (not that it's related, but caught me off guard)

Reply:
This is bizarre, must be some sort of temporary refactoring damage left in by mistake